cli/coverage_cmd.go

```diff
@@ -144,11 +162,11 @@ func coverage(repo *Repo) (*cvg.Coverage, error) {
 		data = append(data, item)
 
 		for _, def := range item.Defs {
-			defKeys[adjustDefKey(def.DefKey, sourceUnit)] = struct{}{}
+			defKeys[def.DefKey] = struct{}{}
 		}
 
 		for _, ref := range item.Refs {
-			ref.SetFromDefKey(adjustDefKey(ref.DefKey(), sourceUnit))
+			ref.SetFromDefKey(ref.DefKey())
```

Comment:
Not needed if we about to drop `adjustDefKey`, the purpose of `ref.SetFromDefKey` was to update ref's data